Week03/lesson06/homework/startup_matchmaker/css/developersstyle.css

```diff
@@ -0,0 +1,187 @@
+/*
+Project Name: Blank Template
+Client: Your Client
+Author: Your Name
+Developer @GA in NYC
+*/
+
+
+/******************************************
+/* SETUP
+/*******************************************/
+
+/* Box Model Hack */
+* {
+ box-sizing: border-box; /* IE */
+}
+
+/* Clear fix hack */
+.clearfix:after {
+  content: ".";
+  display: block;
+  clear: both;
+  visibility: hidden;
+  line-height: 0;
+  height: 0;
+}
+
+.clear {
+ clear: both;
+}
+
+/******************************************
+/* BASE STYLES
+/*******************************************/
+
+html {
+  background-color: #bebebe;
+}
+
+body {
+  line-height: 1.5;
+  font-family: arial, helvetica, sans-serif;
+  font-size: 18px;
+}
+
+/******************************************
+/* LAYOUT
+/*******************************************/
+
+/* Center the container */
+.container {
+  width: 960px;
+  margin: 0 auto;
+}
+
+header {
+  height: 100px;
+  background-color: #000;
+  border-bottom: 6px solid yellow;
+  margin-bottom: 15px;
+}
+
+.logo {
+  display: inline-block;
+  padding-top: 25px;
+}
+
+.logo a {
+  font-size: 30px;
+  text-decoration: none;
+  color: #fff;
+}
+
+nav a {
+  display:inline-block;
+  height: 94px;
+  padding-top: 35px;
+  padding-left: 10px;
+  padding-right: 10px;
+  text-decoration: none;
+  color: #fff;
+  font-family: arial, helvetica, sans-serif;
+  font-size: 18px;
+  vertical-align: top;
+}
+
+
+nav {
+  float: right;
+  vertical-align: top;
+}
+
+nav a:hover {
+  color: yellow;
+}
+
+#active {
+  background-color: grey;
+}
+
+.row {
+  padding-left: 45px;
+}
+
+.column {
+  width: 25%;
+  float: left;
+  padding: 10px;
+  background-color: #fff;
+  margin-right: 15px;
+  margin-bottom: 15px;
+}
+
+.column:first-child {
+  margin-left: -45px;
+}
+
+.column:last-child {
+  margin-right: 0;
+}
+
+footer {
+  height: 75px;
+  background-color: #000;
+  padding-top: 30px;
+}
+
+
+
+/******************************************
+/* OTHER STYLES
+/*******************************************/
+
+h1 {
+  font-family: georgia, serif;
+  font-weight: bold;
+}
+
+h2 {
+  color:#666;
+}
+
+li {
+  display: inline-block;
+  font-size: 12px;
+  padding: 2px 10px;
+  color: #fff;
+  background-color: #666;
+  -webkit-border-radius: 10px;
+  -moz-border-radius: 10px;
+  border-radius: 10px;
+}
+
+
+.boldLink, .button {
+  font-family: oswald, sans-serif;
+  font-weight: 700;
+  letter-spacing: 2px;
+  text-transform: uppercase;
+}
+
+.button {
+  display: block;
+  margin-top: 10px;
+  -webkit-border-radius: 5;
+  -moz-border-radius: 5;
+  border-radius: 5px;
+  -webkit-box-shadow: 0px 3px 0px #bbbb07;
+  -moz-box-shadow: 0px 3px 0px #bbbb07;
```

Comment:
`box-shadow` is supported by all browsers except IE8 basically, so I think you can get rid of the pseudo-styles here

[http://caniuse.com/#search=box-shadow](http://caniuse.com/#search=box-shadow)